package.json

```diff
@@ -19,15 +19,15 @@
     "fable-splitter": "file:./src/js/fable-splitter",
     "fable-utils": "file:./src/js/fable-utils",
     "mocha": "^5.2.0",
-    "rollup": "^0.59.3",
+    "rollup": "^0.59.4",
```

Comment:
Are we still using Rollup? I haven't updated the plugin for Fable 2. I need to try if we can use Webpack 4 and uglify now when compiling FCS + Fable. Fable 2 uses less weird ES2015 features so I hope we've more luck now.

Reply:
I don't know, I still like rollup for it's clean output, but I don't follow who's the winner of the top bundler competition, so you probably know better.

Reply:
The winner is clearly Webpack, there've been many improvements in v4 and it enjoys big momentum now. Rollup still produces a much cleaner output but it's probably not worthy to maintain both plugins at this point. When we want to check the generated JS we can still use fable-splitter. Anyways, don't worry. When we start updating the REPL to Fable 2 I'll remove the dependency :+1: